src/Style.ts

```diff
@@ -1,5 +1,6 @@
 import styled, { createGlobalStyle } from "styled-components";
 
+// eslint-disable-next-line import/prefer-default-export
 export const GlobalStyle = createGlobalStyle`
   * {
     box-sizing: border-box;
```

Comment:
you can also try to use normalize css 
https://necolas.github.io/normalize.css/